plugins/content/caption/base/lib/kCaptionsContentManager.php

```diff
@@ -21,7 +68,26 @@ public function __construct()
 	 * @return string
 	 */
 	public abstract function getContent($content);
-	
+
+	/**
+	 * @param string $content
+	 * @param int $clipStartTime
+	 * @param int $clipEndTime
+	 * @return array
+	 */
+	public abstract function buildFile($content, $clipStartTime, $clipEndTime);
+
+	/**
+	 * @param array $matches
+	 * @param int $clipStartTime
+	 * @param int $clipEndTime
+	 * @return string
+	 */
+	public function createAdjustedTimeLine($matches, $clipStartTime, $clipEndTime)
```

Comment:
Change function to pure abstract and protected since no access from the outside.